src/ensembl/src/content/app/browser/browserHelper.test.ts

```diff
@@ -0,0 +1,157 @@
+import {
+  getChrLocationFromStr,
+  getChrLocationStr,
+  getRegionFieldErrorMessages,
+  getRegionEditorErrorMessages
+} from './browserHelper';
+import { createValidationInfo } from 'tests/fixtures/browser';
+import { RegionValidationResponse } from './browserState';
+import { RegionErrors } from './browserConfig';
+
+describe('browserHelper', () => {
+  describe('getChrLocationFromStr', () => {
+    test('outputs ChrLocation when input received as string', () => {
+      expect(getChrLocationFromStr('10:1-10000')).toStrictEqual([
```

Comment:
use faker

Reply:
I used values so that there is no need define a lot of variables for region, start, end, chrLocation and chrLocationStr. Do we really need these to be random though?

Reply:
random variables are a bit more convincing (they show that things aren't some magic numbers or magic strings), and they might, surprisingly, be easier to read.

`expect(getChrLocationFromString(`${chr}:${start}-${end}`)).toStrictEqual([chr, start, end])` is arguably more meaningful than `toStrictEqual([10, 1, 1000])`.

Reply:
I'll add a function that creates both the tupple and string values into the browser fixtures for this